coolfig/providers.py

```diff
@@ -44,13 +44,15 @@ def get(self, key):
         try:
             with open(path) as fh:
                 return fh.read()
-        except:  # TODO: Handle OSErrors (not found, not readable,...)
-            return NOT_PROVIDED
+        except IOError as e:
+            if e.args[0] == 13:  # Wrong permissions
```

Comment:
I believe `IOError` as an `errno` attribute you can use; you can then also use a constate from here instead of the hardcoded int: https://docs.python.org/2/library/errno.html